packages/katello/katello-repos/katello-repos.spec

```diff
@@ -1,20 +1,11 @@
-%global pulp_release stable
-%global pulp_version 2.21
-%global use_pulp_nightly false
-%if 0%{?rhel} == 7
-%global pulp_enabled 1
-%else
-%global pulp_enabled 0
-%endif
-
 %global pulpcore_version 3.9
 
 %define repo_dir %{_sysconfdir}/yum.repos.d
 %define repo_dist %{dist}
 
 %global prereleasesource nightly
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 3
+%global release 4
 
 Name:           katello-repos
 Version:        4.0
```

Comment:
shouldn't nightly be at 4.1 now?

Reply:
Can I blame a lack of branching procedures in packaging? Looks like there's been an rc release in the 2.4 branch so I'll first submit a PR to bump to 4.1 and then rebase this.